atlp-rwanda/phantom-fe-dream-team
# seting up trailwind-css

setting up trailwind-css
How to test it manually 
-clone the repo
- run npm run dev:start in your teminal
-copy http://localhost:3000/
-then go to your browser and paste it there and see how trialwind css looks
